Resources/Prototypes/Loadouts/Jobs/Security/corpsman.yml

```diff
@@ -58,7 +58,33 @@
     - ClothingBeltCorpsmanWebbingFilled
 
 # Ears
+- type: loadout
+  id: LoadoutClothingEyesGlassesMedSec
+  category: JobsSecurityCorpsman
+  cost: 3
+  exclusive: true
+  requirements:
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutSecurityEyes
+    - !type:CharacterJobRequirement
+      jobs:
+        - Brigmedic
+  items:
+    - ClothingEyesGlassesMedSec
 
+- type: loadout
+  id: LoadoutClothingEyesHUDMedSec
+  category: JobsSecurityCorpsman
+  cost: 0
```

Comment:
tbh this is easily can be made by corpsman. this can make this round start easily. like (sechud, sec radio) from secvend. and medhud from nanovend. and wire, capacitor on that one machine i forgot.

beside they are suppose to be healer for sec so having medsec hud easily arent that bad.

Reply:
but il up the price to one just to satified ya